Corona-Warn-App/src/main/java/de/rki/coronawarnapp/dccreissuance/ui/success/DccReissuanceSuccessFragment.kt

```diff
@@ -1,6 +1,25 @@
 package de.rki.coronawarnapp.dccreissuance.ui.success
 
+import android.os.Bundle
+import android.view.View
 import androidx.fragment.app.Fragment
 import de.rki.coronawarnapp.R
+import de.rki.coronawarnapp.databinding.FragmentDccReissuanceSuccessBinding
+import de.rki.coronawarnapp.util.ui.popBackStack
+import de.rki.coronawarnapp.util.ui.viewBinding
 
-class DccReissuanceSuccessFragment : Fragment(R.layout.fragment_dcc_reissuance_success)
+class DccReissuanceSuccessFragment : Fragment(R.layout.fragment_dcc_reissuance_success) {
```

Comment:
we have a nice transition in other result screens can we follow the same here too ? see `DccValidationPassedFragment` or `DccTicketingValidationResultFragment`